lib/mappers/index.js

```diff
@@ -1,13 +1,30 @@
-const mapTextStyle = require('./textstyle');
-const mapColor = require('./color');
-const mapGradient = require('./gradient');
-const mapShadow = require('./shadow');
-const mapBorder = require('./border');
-
-module.exports = {
-  mapColors: colors => colors.map(mapColor),
-  mapTextStyles: textStyles => textStyles.objects.map(mapTextStyle),
-  mapGradients: gradients => gradients.map(mapGradient),
-  mapShadows: shadows => shadows.map(mapShadow),
-  mapBorders: borders => borders.map(mapBorder),
+const mapTextStyleSketch = require('./sketch/textstyle');
+const mapColorSketch = require('./sketch/color');
+const mapGradientSketch = require('./sketch/gradient');
+const mapShadowSketch = require('./sketch/shadow');
+const mapBorderSketch = require('./sketch/border');
+
+const mapTextStyleFigma = require('./figma/textstyle');
+const mapColorFigma = require('./figma/color');
+const mapGradientFigma = require('./figma/gradient');
+const mapShadowFigma = require('./figma/shadow');
+const mapBorderFigma = require('./figma/border');
+
+module.exports = extension => {
+  const ex = extension.toLowerCase();
+
+  module.mapColors = colors =>
```

Comment:
I would never reference global namescapes unless applying a hack or workaround. Is there a reason why you would do this?

Reply:
Except my lack of native javascript programming, probably not :p